dts/bindings/spi/st,stm32h7-spi.yaml

```diff
@@ -12,3 +12,18 @@ description: |
 compatible: "st,stm32h7-spi"
 
 include: st,stm32-spi-common.yaml
+
+properties:
+  midi-clock:
+    type: int
+    default: 0
+    description: |
+      Only for STM32H7, (Master Inter-Data Idleness) minimum
```

Comment:
Restriction `Only for STM32H7,` is misleading. This property is made available on `st,stm32h7-spi` compatible, hence will be available for compatible SoCs: H5/H7/MP1/U5/WBA. This is perfectly fine and sufficient

Reply:
good point, i did update document :)